lib/config-helper.js

```diff
@@ -0,0 +1,360 @@
+
+var fs = require('fs-extra');
+var pathExtra = require('path-extra');
+var configWatcher = require('./config-watcher');
+
+module.exports = function(){
+
+  var config;
+  var home = '';
+  var mainAppHost = 'localhost';
+  var mainAppPort = 19104;
+  var appsPort = 18001;
+  //var loadedPlugins = {};
```

Comment:
hm not so simple. This is a regression to manage. Actually the export function should be moved to application / plugin helpers. So it s commented to leave a mark.